libutils/mustache.c

```diff
@@ -652,6 +764,28 @@ static bool Render(Buffer *out, const char *start, const char *input, Seq *hash_
                     switch (JsonGetContainerType(var))
                     {
                     case JSON_CONTAINER_TYPE_OBJECT:
+                        {
+                            if (!ShouldIterateObject(input, delim_start, *delim_start_len, delim_end, *delim_end_len))
+                            {
+                                const char *cur_section_end = NULL;
+
+                                if (!Render(out, start, input,
+                                            hash_stack,
+                                            NULL,
+                                            delim_start, delim_start_len, delim_end, delim_end_len,
+                                            skip_content || tag.type == TAG_TYPE_INVERTED, section, &cur_section_end))
+                                {
+                                    free(section);
+                                    return false;
+                                }
+                                input = cur_section_end;
+                                free(section);
+                                break;
+                            }
+                            /* else fall through to the case below because
+                             * iterated objects and arrays are processed in the
+                             * same way */
+                        }
```

Comment:
Ugh, I'm getting dizzy.

Reply:
Yeah, cannot blame you. I'm still recovering from trying to figure out WTH is going on in this crazy giant function.

Reply:
MVR